net/http/gzip/gzip_test.go

```diff
@@ -26,7 +26,7 @@ func (n noOpWriter) Write(d []byte) (int, error) {
 func (n noOpWriter) WriteHeader(int) {}
 
 func BenchmarkGzipSmall(b *testing.B) {
-	r, _ := http.NewRequest("GET", "/foo", nil)
+	r, _ := http.NewRequest("GET", "/foo", nil) // #nosec
```

Comment:
I'm not sure why `-skip="*_test.go"` failed to prevent this file from getting scanned (it seems to be working successfully for several other tests). Possibly a bug in gas?